### ISSUE TITLE
Clean SD Card data

### ISSUE THREAD

**Issue by bgeorges** [bgeorges/safecast#1]
We want a program that takes as input the CSV data from the bGeigie Nano and that remove the entries with no GPS coordinates.

**Comment by Alopez63** [comment]
Can u give me more details, what will be a wrong entry and what will be the correct version.

**Comment by Alopez63** [comment]
Found it. The lines that have a valid GSP Fix. See: https://github.com/Safecast/bGeigieNanoKit/wiki/Operations-Manual#data-log
    # 'A' = Available or OK, 'V' = Void or invalid
Can you confirm.

**Comment by bgeorges** [comment]
That's correct. The output should only contain those records which are marked as A ("Available or OK")
